src/Forside/barnehage/SammenligningMedBransje/EkspanderbartSammenligningspanel.tsx

```diff
@@ -65,6 +70,9 @@ export const EkspanderbartSammenligningspanel: FunctionComponent<Props> = ({
 
```

Comment:
Vil vi også har TODO i GetPanelEventTekst(...)
for case SammenlingingsType.GRADERT ?